test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -134,29 +143,27 @@ class test_ucp_ep_reconfig : public ucp_test {
 #endif
 
         for (auto msg_size : msg_sizes) {
-            for (unsigned i = 0; i < num_iterations; ++i) {
-                mem_buffer sbuf(msg_size, UCS_MEMORY_TYPE_HOST, i);
-                mem_buffer rbuf(msg_size, UCS_MEMORY_TYPE_HOST, ucs::rand());
-                mem_buffer o_sbuf(msg_size, UCS_MEMORY_TYPE_HOST, i);
-                mem_buffer o_rbuf(msg_size, UCS_MEMORY_TYPE_HOST, ucs::rand());
+            mem_buffer_vec_t sbufs, rbufs, o_sbufs, o_rbufs;
+            std::vector<void*> reqs;
 
-                std::vector<void*> reqs;
-                send_message(sender(), receiver(), sbuf, rbuf, msg_size, reqs);
-
-                if (bidirectional) {
-                    send_message(receiver(), sender(), o_sbuf, o_rbuf, msg_size,
-                                 reqs);
-                }
+            init_buffers(sbufs, rbufs, o_sbufs, o_rbufs, msg_size);

```

Comment:
Maybe init them outside the loop, only once?

Reply:
will be wrong as they are sent and verified on each iteration